controllers/core/openstackcontrolplane_controller.go

```diff
@@ -85,6 +86,7 @@ type OpenStackControlPlaneReconciler struct {
 //+kubebuilder:rbac:groups=nova.openstack.org,resources=nova,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=mariadb.openstack.org,resources=mariadbs,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=neutron.openstack.org,resources=neutronapis,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=redhat.com,resources=openstackansibleees,verbs=get;list;watch;create;update;patch;delete
```

Comment:
@jlarriba @slagle Ansible EE group `redhat.com` while the group for the other operators are `openstack.org`

should we change it to follow the pattern?

Reply:
Yes

Reply:
https://github.com/openstack-k8s-operators/openstack-ansibleee-operator/pull/56